src/functions-and-arrays.js

```diff
@@ -1,19 +1,116 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 >= num2) {
+    return num1;
+  } else if (num1 <= num2) {
+    return num2;
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arr) {
+  if (arr.length === 0) {
+    return null
+  } else if (arr.length === 1) {
+    return arr[0];
+  } else {
+    let longestWord = '';
+    for (let i = 0; i < arr.length; i++) {
+      if (arr[i].length > longestWord.length) {
+        longestWord = arr[i]
+      }
+    }
+    return longestWord;
+  }
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(arr) {
+  if (arr.length === 0) {
+    return 0;
+  } else {
+    let sum = 0;
+    for (let i = 0; i < arr.length; i++) {
+      sum += arr[i];
+    }
+    return sum;
+  }
+}
+
+function sum(arr) {
```

Comment:
Perhaps the name of this function could be more descriptive: the sum of what will be returned?